fcsparser/api.py

```diff
@@ -430,6 +444,12 @@ def reformat_meta(self):
         meta['_channels_'] = df
         meta['_channel_names_'] = self.get_channel_names()
 
+    # Constructs Pandas dataframe
+    def dataframe(self):
```

Comment:
@eyurtsev Sure, I like the idea of a property the best, but again, I'm fine either way.